app.js

```diff
@@ -37,9 +56,14 @@ export default class FlashCards extends Component {
 
   render() {
     return(
-      <div className="container">
-        <Form value={this.state} change={this.handleChange} submit={this.handleSubmit}/>
+      <div className="vertical-center">
+        <div className="container">
+          <Nav />
+          <h1 className="title text-center">React Flash Cards <i className="text-primary fab fa-react"></i></h1>
+          <View data={this}/>
```

Comment:
This is a big no-no. Unless you are authoring a really complicated React component library, there is never a reason to pass a whole component instance to another component. Only pass the component what it needs. This is the reason I recommend removing your `View` component. You can add a `renderView` method to your `App` instead.